packages/components/src/components/Slider/__tests__/Slider.ct.tsx

```diff
@@ -0,0 +1,227 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { expect, type MountResult, test } from '@playwright/experimental-ct-react';
+import { type Locator, type Page } from '@playwright/test';
+import sinon from 'sinon';
+
+import { Slider } from '../Slider';
+
+const SLIDER_TEST_ID = 'test-slider';
+const ARIA_LABEL = 'test slider';
+
+async function dragSlider(
+    page: Page,
+    container: MountResult | Locator,
+    slider: Locator,
+    travelPercentage: number,
+    onBeforeMouseUp?: () => void,
+) {
+    const sliderBox = await slider.boundingBox();
+
+    const containerBox = await container.boundingBox();
+
+    if (!sliderBox || !containerBox) {
+        throw new Error('Slider position not found');
+    }
+
+    const startingXPos = sliderBox.x + sliderBox.width / 2;
+    const travelDistance = containerBox.width * (travelPercentage / 100);
+    const endingXPos = startingXPos + travelDistance;
+    const startingYPos = sliderBox.y + sliderBox.height / 2;
+
+    await slider.hover();
+    await page.mouse.down();
+    await page.mouse.move(endingXPos, startingYPos);
+    onBeforeMouseUp && onBeforeMouseUp();
+    await page.mouse.up();
+}
+
+test('should render a slider', async ({ mount }) => {
+    const component = await mount(<Slider data-test-id={SLIDER_TEST_ID} aria-label={ARIA_LABEL} defaultValue={[0]} />);
+    await expect(component).toBeVisible();
+    await expect(component.getByRole('slider')).toBeVisible();
+});
```

Comment:
you are testing this case with all other tests, I would remove
```suggestion
```

Reply:
true